examples/container/resource/resource_user/manifest.yaml

```diff
@@ -0,0 +1,10 @@
+name: resource_user
+version: 0.0.1
+init: /resource_user
+mounts:
+  /lib:
+    host: /lib
+  /lib64:
+    host: /lib64
```

Comment:
For now we have in all the manifests a `/system` bind mount to `/system` to be unable on Android. We should this here as well until we have a better solution.